frontend/gatsby-node.js

```diff
@@ -34,4 +34,40 @@ exports.createPages = ({ actions }) => {
             isPermanent: true
         });
     }
+
+    // Generate VR pages
+    const vrImageTemplate = require.resolve(
+        `./src/templates/vrImageTemplate.js`
+    );
+    const result_vr = await graphql(`
+        {
+            allFile(filter: { relativeDirectory: { eq: "vr-photo-spheres" } }) {
+                edges {
+                    node {
+                        name
+                        publicURL
+                        relativePath
+                    }
+                }
+            }
+        }
+    `);
+    // Handle errors
+    if (result_vr.errors) {
+        reporter.panicOnBuild(`Error while running GraphQL query.`);
```

Comment:
you could probably mention that it's related to the VR pages.

Reply:
Resolved